charts/sophora-server/templates/statefulset.yaml

```diff
@@ -331,6 +331,12 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.serverModeLabeler.image.repository }}:{{ .Values.serverModeLabeler.image.tag }}"
           imagePullPolicy: {{ .Values.serverModeLabeler.image.pullPolicy }}
+          command:
```

Comment:
There is no need to overwrite the `command`. You can fully configure the tool using environment variables. The missing one is `SERVER_URL` .

Reply:
Good to know! That makes everything easier.